source/irfs/irf_components/index.rst

```diff
@@ -104,7 +109,7 @@ Probability distributions
   This is the canonical form we use and the values we store in files.
 * Often, when comparing observered event distributions with a PSF model,
   the :math:`dP/dr^2` distributions in equal-width bins in :math:`r^2` is
-  used. The relation is :math:`d\Omega = \pi dr^2`, i.e. :math:`dP/dr^2=(1/\pi)(dP/d\Omega)`.
+  used. The relation is :math:`d\Omega = \pi dr^2`, i.e. :math:`dP/d\Omega=(1/\pi)(dP/dr^2)`.
```

Comment:
Just to point it out, since it's non-obvious: there was an error here in the relation between dP/dOmega and dP/dr^2, fixed by @lmohrmann in this PR. I have double checked that now all equations for the PSF are correct.